app/src/main/java/com/example/xpvehicles/activities/RentingRequestsActivity.java

```diff
@@ -0,0 +1,163 @@
+    private TextView tvOrderSummaryOrderTotal;
+    private TextView tvRentingRequestsPickUpDate;
+    private TextView tvRentingRequestReturnDate;
+    private TextView tvRentingRequestStatus;
+
+    private void setTopAppBarOnClickListener() {
+        topAppBarRentingRequest.setNavigationOnClickListener(v -> {
+            this.finish();
+        });
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_renting_requests);
+        rentVehicle = getIntent().getParcelableExtra("rentVehicle");
+        originalVehicle = (Vehicle) rentVehicle.getVehicle();
+        bind();
+        setTopAppBarOnClickListener();
+        try {
+            bindVehicleImagesAdapter();
+            setValues();
+        } catch (ParseException e) {
+            Log.e(TAG, "error setting the values for RentingRequestsActivity");
+        }
+    }
+
+    private void bindVehicleImagesAdapter() throws ParseException {
+        List<ParseFile> images = originalVehicle.getVehicleImages();
+        ViewPager2 viewPager = findViewById(R.id.viewPagerRentingRequestVehicleImages);
+        VehicleImagesAdapter vehicleImagesAdapter = new VehicleImagesAdapter(this, images);
+        viewPager.setAdapter(vehicleImagesAdapter);
+        setVehicleSwipeListener(viewPager, images.size());
+    }
+
+    private void setVehicleSwipeListener(ViewPager2 viewPager, int totalNumberOfImages) {
+        viewPager.registerOnPageChangeCallback(new ViewPager2.OnPageChangeCallback() {
+            @Override
+            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+                super.onPageScrolled(position, positionOffset, positionOffsetPixels);
+                TextView tvVehicleImagePosition = findViewById(R.id.tvVehicleImagePosition);
+                int currentPosition = position + 1;
+                tvVehicleImagePosition.setText(currentPosition + " / " + totalNumberOfImages);
+            }
+        });
+    }
+
+    private void bind() {
+        topAppBarRentingRequest = findViewById(R.id.topAppBarRentingRequest);
+        tvRentingRequestVehicleName = findViewById(R.id.tvRentingRequestVehicleName);
+        tvRentingRequestVehicleDescription = findViewById(R.id.tvRentingRequestVehicleDescription);
+        tvRentingRequestDailyPrice = findViewById(R.id.tvRentingRequestDailyPrice);
+        tvOrderSummaryDailyPrice =  findViewById(R.id.tvOrderSummaryDailyPrice);
+        tvOrderSummaryNumberOfDays =  findViewById(R.id.tvOrderSummaryNumberOfDays);
+        tvOrderSummaryOrderTotal =  findViewById(R.id.tvOrderSummaryOrderTotal);
+        tvRentingRequestsPickUpDate =  findViewById(R.id.tvRentingRequestsPickUpDate);
+        tvRentingRequestReturnDate =  findViewById(R.id.tvRentingRequestReturnDate);
+        tvRentingRequestStatus =  findViewById(R.id.tvRentingRequestStatus);
+    }
```

Comment:
1. I wouldn't call this bind(). Bind is binding data to views, but you are initializing views in here.
2. There is extra space after '='

Reply:
I call it bind, just to be consistent with adapters